src/layouts/BaseLayout.tsx

```diff
@@ -25,6 +25,8 @@ export type LayoutProps = {
   showBallot?: boolean;
   showInfo?: boolean;
   showSubmitButton?: boolean;
+  marginTop?: boolean;
+  paddingX?: boolean;
```

Comment:
It's not a good practice to pass single style props. Better to pass `className` using `clsx`